clearwater-memcached/usr/share/clearwater/infrastructure/scripts/memcached

```diff
@@ -25,4 +25,6 @@ sed -e 's/^-l .*$/-l '$listen_address'/g'\
     -e 's/^# *-v *$/-v/g'\
     -e 's/^\(# *\|\)-c.*$/-c 4096/g' </etc/memcached.conf >/etc/memcached_11211.conf
 
+echo "\n# Disable listening on UDP\n-U 0" >> /etc/memcached_11211.conf
+
```

Comment:
@richardwhiuk, is there a reason to use `echo -e`, over just using `printf` without changing to bash? I've tested `printf` and it does the right thing.